common/character_interactions/00_alliance.txt

```diff
@@ -97,6 +97,11 @@
 				}
```

Comment:
It says I attempted murder.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/117038969-4a768100-ad19-11eb-8948-165a6efbfa6f.png)

</details>

Reply:
As far as I can tell all hostile schemes give the same modifier when discovered, check `hostile_scheme_discovery.3003`

Reply:
> As far as I can tell all hostile schemes give the same modifier when discovered, check `hostile_scheme_discovery.3003`

Time to change it. You added new scheme after all.